.github/workflows/cargo_build.yml

```diff
@@ -20,6 +20,9 @@ on:
         required: true
         type: string
 
+env:
+  OPENSSL_DIR: /tmp/openssl_fips
```

Comment:
Are we sure it is not going to install at this location where, for some reason, another installation would still be ?
I used `$(mktemp -d)` to avoid this but it's probably not necessary.